src/metatrain/experimental/pet/trainer.py

```diff
@@ -256,9 +256,9 @@ def train(
             else:
                 pet_model.hypers.TARGET_TYPE = "structural"
                 pet_model.TARGET_TYPE = "structural"
-            pet_model = pet_model.to(device=device, dtype=dtype)
         else:
             pet_model = PET(ARCHITECTURAL_HYPERS, 0.0, len(all_species))
+        pet_model = pet_model.to(device=device, dtype=dtype)
```

Comment:
This should be already merged, right?

Reply:
This should be gone now that I've updated the branch